package.json

```diff
@@ -41,7 +41,7 @@
     "sudo-prompt": "^8.2.3",
     "susie": "^3.0.0",
     "ts-node": "^8.10.1",
-    "typescript": "^3.8.3",
+    "typescript": "^3.9.10",
```

Comment:
would it make sense to use `~` for `typescript` given we're pinning it as well and it doesn't follow semver strictly?

Reply:
yep i think so 👍